src/azure-cli-core/setup.py

```diff
@@ -66,6 +66,10 @@
     'azure-common~=1.1',
 ]
 
+# dependencies for specific OSes
+if not sys.platform.startswith('cygwin'):
+    DEPENDENCIES.append('psutil~=5.7')
+
```

Comment:
Move the dependency of `psutil` to core as it is used by core, not command modules. @houk-ms